official/docs/csharp/pickups/create-pickup.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using EasyPost;
+
+namespace EasyPostExamples
+{
+    public class Examples
+    {
+        public static async Task Main()
+        {
+            string apiKey = Environment.GetEnvironmentVariable("EASYPOST_API_KEY")!;
+
+            EasyPost.ClientManager.SetCurrent(apiKey);
+
+            Pickup pickup = await Pickup.Create(new Dictionary<string, object>()
+            {
+                {
+                    "reference", "my-first-pickup"
+                },
+                {
+                    "min_datetime", "2022-10-01 10:30:00"
+                },
+                {
+                    "max_datetime", "2022-10-01 17:30:00"
```

Comment:
```suggestion
                    "max_datetime", "2022-10-02 10:30:00"
```
should be the next day, to be consistent with other languages code example